src/GoodsList.tsx

```diff
@@ -8,7 +8,7 @@ type Props = {
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li key={good.id} data-cy="good" style={{ color: good.color }}>
```

Comment:
use classnames instead style, inline styles it is bad practice

https://blog.logrocket.com/why-you-shouldnt-use-inline-styling-in-production-react-apps/